packages/block-library/src/post-featured-image/index.php

```diff
@@ -103,8 +103,8 @@ function get_block_core_post_featured_image_border_attributes( $attributes ) {
 
 	$border_attributes = gutenberg_style_engine_get_styles( array( 'border' => $border_styles ) );
 	return array(
-		'class' => $border_attributes['classnames'],
-		'style' => $border_attributes['css'],
+		'class' => array_key_exists( 'classnames', $border_attributes ) ? $border_attributes['classnames'] : '',
```

Comment:
Sounds fine to me.

Though looking at other uses of `gutenberg_style_engine_get_styles` I see that all consumers following this pattern instead:

```php
$attributes = array();
$styles = gutenberg_style_engine_get_styles();

if ( ! empty( $styles['classnames'] ) ) {
    $attributes['class'] = $styles['classnames'];
}

if ( ! empty( $styles['css'] ) ) {
    $attributes['style'] = $styles['css'];
}

return $attributes;
```

Reply:
Updated per your suggestion, thanks!